src/mlpack/methods/neighbor_search/ns_model_impl.hpp

```diff
@@ -484,6 +492,10 @@ std::string NSModel<SortPolicy>::TreeName() const
       return "R++ tree";
     case VP_TREE:
       return "Vantage point tree";
+    case RP_TREE:
+      return "Random projection tree (mean split)";
```

Comment:
No need for a capital r here, lowercase fits better with how these strings are used (I guess that applies to the VP tree too).